lpc55_sign/src/signed_image.rs

```diff
@@ -359,31 +139,19 @@ pub fn create_cmpa(
     with_dice_cust_cfg: bool,
     with_dice_inc_sec_epoch: bool,
     rkth: &[u8; 32],
-    cmpa_path: &Path,
-) -> Result<()> {
-    let mut cmpa_out = OpenOptions::new()
-        .write(true)
-        .truncate(true)
-        .create(true)
-        .open(cmpa_path)?;
-
+) -> Result<Vec<u8>> {
```

Comment:
Could we return the `CMPAPage` here to keep it strongly typed for longer?

Reply:
Yep, done as part of a8b5677.